packages/fxa-payments-server/src/lib/AppContext.tsx

```diff
@@ -20,6 +21,7 @@ export const defaultAppContext = {
   config,
   getScreenInfo: () => new ScreenInfo(),
   locationReload: noopFunction,
+  matchMedia: window.matchMedia || function() { return { matches: false }},
```

Comment:
Don't refer to `window` here, just keep that isolated to `src/index.tsx` in the wrapper function. This default shouldn't get used in practice - both the app and tests should be providing a value, so `noopFunction` should work here.

Reply:
noopFunction causes typescript to fail in this case :thinking:

Reply:
Hmm, also looks like `checkMedia` doesn't need the whole `MediaQueryList` object. Could just return `matches` from the wrapper in `src/index.tsx` and not have to cover the whole `MediaQueryList` object in types & mocks.